src/launcher.c

```diff
@@ -455,6 +455,19 @@ static int load_fonts(char *fonts_path)
               sdk_version == SDK_API_LEVEL_14)) {
     load_addr = STAX_FONTS_ARRAY_ADDR;
     load_size = 40960;
+  } else if (sdk_version == SDK_API_LEVEL_15) {
```

Comment:
could be simplified if we merge https://github.com/LedgerHQ/speculos/pull/448

Reply:
PR merged :)

Reply:
I will modify accordingly